robottelo/utils/issue_handlers/README.md

```diff
@@ -1,87 +1,139 @@
-# Issue tracker handlers
+# Jira Issue Handler
```

Comment:
I'd recommend deleting this file and moving:
1. API doc to `__doc__` of the individual classes, functions and methods. If we need to generate an actual doc from it, then we should use Sphinx for that, which we do, but we do not publish the result of the docs build anywhere.
2. And moving the general info about Issue Handlers to the robottelo wiki. That makes sure that we keep documentation in one place.

Reply:
Removed the README and updated the doc strings for individual classes, functions, and methods.